backend/database/migrations/2021_05_18_070618_create_addfriends_table.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateAddfriendsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('addfriends', function (Blueprint $table) {
+            $table->id();
+            $table->integer('user1');
+            $table->integer('user2');
+            $table->integer('status')->default(0);
+            $table->timestamps();
+        });
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::dropIfExists('addfriends');
+    }
+}
```

Comment:
add newline

Reply:
was this resolved? cant see atm

Reply:
i still see it: 
![image](https://user-images.githubusercontent.com/72127499/118642649-e5706000-b7db-11eb-970a-b1cbf0a2c305.png)

Reply:
@ichbinjaeger good catch, it's fixed